tests/phpunit/FunctionalAbsTest.php

```diff
@@ -142,9 +142,70 @@ public function testRestoreTriggersDisabled(): void
         $this->assertEmpty($errorOutput);
     }
 
+    public function testRestoreBucketsDisabled(): void
+    {
+        $this->createConfigFile(
```

Comment:
Tady (a v dalších voláních `createConfigFile()`) by asi byly přehlednější pojmenované argumenty), co myslíš?

Reply:
no tak určitě... hlavně všechny ty testy by chtěly úplně předělat